tango_simlib/utilities/helper_module.py

```diff
@@ -81,6 +81,21 @@ def get_server_name():
     server_name = executable_name + '/' + sys.argv[1]
     return server_name
 
+def get_file_name():
+    """Gets the db file name from the command line arguments
+
+    Returns
+    -------
+    file_name : str
+        file used as tango database
+    """
+    args = sys.argv
+    for index, val in enumerate(args):
+        if 'file' in val:
```

Comment:
This is much better.  Could still fail if  someone starts a device like "fileServer".  About the strictest we can make it is `startswith('-file=')`

Reply:
Yes. Better suggestion. Thanks.